django/db/backends/base/operations.py

```diff
@@ -46,6 +46,9 @@ class BaseDatabaseOperations:
     UNBOUNDED_FOLLOWING = 'UNBOUNDED ' + FOLLOWING
     CURRENT_ROW = 'CURRENT ROW'
 
+    # Prefix for EXPLAIN queries. None if the backend does not support this.
```

Comment:
`None` should be Empty (or Empty string)?

Reply:
👍